recipely-server/config/config.js

```diff
@@ -0,0 +1 @@
+module.exports = '6a637c79023fc91e5eebbd087157b560';
```

Comment:
Also, create a "config.example.js" that shows how the config file should look. The "config.example.js" will be committed to GitHub, and not "config.js".